src/common/hist_util.cc

```diff
@@ -552,6 +552,7 @@ void BuildHistDenseKernel(const std::vector<GradientPair>& gpair,
   const BinIdxType* gradient_index = gmat.index.data<BinIdxType>();
```

Comment:
I think there's no actual change in this file right?

Reply:
yes, reverted